postgres/candy_friends.py

```diff
@@ -0,0 +1,21 @@
+
+def show_favorite_candy():
+    import psycopg2
+    try:
+        conn = psycopg2.connect("dbname='adhoc_db' user='postgres' host='localhost' password='RustyLemon'")
+    except:
+        print("I am unable to connect to the database")
```

Comment:
This try/except isn't very helpful, I'd suggest removing it or better catch a specific error and provide valuable information.

Bare excepts without a specific exception are almost never okay.